### DISCUSSION TITLE
[formControl] isn't a property of <tui-input-file>

### DISCUSSION THREAD

**Discussion by GaryTimeless** [taiga-family/taiga-ui#4399] [Q&A] [answered]
Dear community, 

I am a student, working on a new Project. 
First step is to upload a XML file.
I ran into your beautiful UI Framework and wanted to use
https://taiga-ui.dev/components/input-files/Setup

while Copy Pasting - I always get this error. 
_Can't bind to 'formControl' since it isn't a known property of 'tui-input-file'._
I don't have any special things going on.
Basic angular app
first component -> upload.component.html

copy paste -> Error.

any idea?

Oh, chatGPT suggests 

<tui-input-files
  *ngIf="!control.value"
  accept="image/*"
  (filesChange)="onFilesChange($event)"
  (reject)="onReject($event)"
></tui-input-files>

<img width="2432" alt="image" src="https://user-images.githubusercontent.com/60399535/236574562-86c837a8-da48-4d89-a542-09830caa41c6.png">

**Comment by waterplea** [comment, marked as answer]
You need to import `ReactiveFormsModule` to use Angular forms.
